client/index.html

```diff
@@ -152,11 +150,6 @@
 
 <script src="client.js"></script>
 
-<body onresize="on_window_resize()">
-
-
-
-
-</body>
+<!-- <body onresize="on_window_resize()"></body> -->
+<body></body>
```

Comment:
(I only tested in Chrome)

The `on_window_resize` process was already broken, but after fixing it, the experience in the frontend is actually worse, so I've commented out the resize function until I understand what the original intent was. Commenting out this resize function makes the frontend more usable.

For me, `on_window_resize` causes all content in the consoles to be reduced to just one character.